src/main/java/org/zalando/nakadi/service/SchemaService.java

```diff
@@ -35,4 +45,21 @@ public SchemaService(final SchemaRepository schemaRepository,
                         () -> schemaRepository.getSchemasCount(name)));
     }
 
+    public Result<EventTypeSchema> getSchemaVersion(final String name, final String version) {
+        final Matcher versionMatcher = VERSION_PATTERN.matcher(version);
```

Comment:
https://github.com/zalando/nakadi/commit/d1aea1cf333888fe78b40bbf7cc37a80c8909ad7 fixes the status code but doesn't use Spring validation (yet)